examples/run/alpaka/CMakeLists.txt

```diff
@@ -4,21 +4,42 @@
 #
 # Mozilla Public License Version 2.0
 
-set(LIBRARIES vecmem::core traccc::io traccc::performance
-    traccc::core traccc::device_common traccc::alpaka alpaka::alpaka
-    traccc::options)
+set(EXTRA_LIBS "")
 
 if(alpaka_ACC_GPU_CUDA_ENABLE)
   enable_language(CUDA)
   include( traccc-compiler-options-cuda )
 
-  list (APPEND LIBRARIES vecmem::cuda traccc::cuda)
+  list (APPEND EXTRA_LIBS vecmem::cuda traccc::cuda)
 elseif(alpaka_ACC_GPU_HIP_ENABLE)
   enable_language(HIP)
   find_package( HIPToolkit REQUIRED )
-  list(APPEND LIBRARIES HIP::hiprt vecmem::hip)
- endif()
 
+  list(APPEND EXTRA_LIBS HIP::hiprt vecmem::hip)
+endif()
+
+set(LIBRARIES vecmem::core traccc::io traccc::performance
+    traccc::core traccc::device_common traccc::alpaka alpaka::alpaka
+    traccc::options ${EXTRA_LIBS})
+
+traccc_add_executable( seq_example_alpaka "seq_example_alpaka.cpp"
+    LINK_LIBRARIES ${LIBRARIES} )
 traccc_add_executable( seeding_example_alpaka "seeding_example_alpaka.cpp"
     LINK_LIBRARIES ${LIBRARIES} )
 
+# #
```

Comment:
PR #572 added track fitting and finding to the throughput examples, which the Alpaka code base doesn't support yet, breaking that bit of this PR.

I can either remove all the code, or leave it as I have done here with that code not compiling and it can just be reactivated when I add the track finding and fitting bits (which is in progress and about 80% of the way there).